README.md

```diff
@@ -507,6 +514,75 @@ For more information about `vfc-vtk.py`, please use the online help:
 $ postprocess/vfc-vtk.py --help
 ```
 
+## Pinpointing errors with Delta-Debug
+
+[Delta-Debug (DD)](https://www.st.cs.uni-saarland.de/dd/) is a general bug
+reduction method that allows to efficiently find a minimal set of conditions
+that trigger a bug. In this case, we are going to consider the set of
+floating-point instructions in the program. We are using DD to find a minimal
+set of instructions responsible for the instability in the output.
+
+By testing instructions sub-sets and their complement, DD is able to find
+smaller failing sets step by step. DD stops when it finds a failing set where
+it cannot remove any instruction. We call such a minimal set _ddmin_.  The
+Delta-Debug implementation for stochastic arithmetic we use here has been
+developed in the [verrou](https://github.com/edf-hpc/verrou) project.
+
+To use delta-debug, we need to write two scripts:
+
+   - A first script ``ddRun <output_dir>``, is responsible for running the
+     program and writing its output inside the ``<output_dir>`` folder.
+
+   - A second script ``ddCmp <reference_dir> <current_dir>``, takes as
+     parameter two folders including respectively the outputs from a reference
+     run and from the current run. The `ddCmp` script must return with a
+     success when the deviation between the two runs is acceptable, and fail if
+     the deviation is unacceptable.
+
+To decide if a given set is unstable, DD will repeat the experiment by running
+the program five times (the number of times can be changed by setting the
+environment variable ``INTERFLOP_DD_NRUNS``).
+
+``ddRun`` and ``ddCmp`` depend on the user's application and the error
+tolerance of the application domain; therefore it is hard to provide a generic
+script that fits all cases. That is why we require the user to manually write
+these scripts.  Once the scripts are written, the Delta-Debug session is
+launched using the following command:
+
+```bash
+$ VFC_BACKENDS="libinterflop_mca.so -p 53 -m mca" vfc_ddebug ddRun ddCmp
+```
+
+The `VFC_BACKENDS` variable selects the noise model among the available
```

Comment:
This is a bit ambiguous on whether the rest of the back-ends can be used as well.
Maybe specify it explicitly?